data_storage/export_format.md

```diff
@@ -5,7 +5,7 @@ Export Format
   working directory, with the file name `<profile_name>.tox`
 
 - **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+  under the `avatars/` directory in the root of the export directory.
```

Comment:
"Export directory" is whatever directory the client decides to export to, it's the directory that will contain the <profile_name>.tox and the avatars/ folder, but I intend to not specify where it should be, since users may want to export to different locations (USB key, config directory, backup location, ...)